Examples/Detectors/DD4hepDetector/include/ActsExamples/DD4hepDetector/GeometryIdentifierHooks.hpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+
+/// Geometry identifiers hooks to be used with DD4Hep detectors
+/// to had some extra identifier to sensitives surfaces.
+namespace det {
```

Comment:
Is this a new namespace?

Reply:
Should probably not be 'set' then, I suppose, if it is really specific to the ODD, I would even call it `ODD::` ...

Reply:
It is not a new namespace, it is also used by DetUtils. I could change it but if I rewrite this to be a generic layerspliter, I guess it would be fine keeping it as is.